### PR TITLE
sdl_rtf: brew audit --strict

### PR DESCRIPTION
**Pull request by L2G**

### CODE REVIEW

File: Library/Formula/sdl_rtf.rb [hunk 1]
```diff
@@ -1,16 +1,19 @@
...
   def install
     system "./configure", "--prefix=#{prefix}"
-    system "make install"
+    system "make", "install"
+  end
+
+  test do
+    # Would need to write a sample C file that can be linked to libSDL_rtf...
+    system "false"
   end
```
Comment (jacknagel) [inline Library/Formula/sdl_rtf.rb]: If we're not actually adding a test at this point then this should be removed.

**Comment by tdsmith** [comment]
Would you be willing to write a test?

**Comment by L2G** [comment]
With all due respect, no. :smile_cat:

**Comment by xu-cheng** [comment]
Again, we don't merge style only change.

**Comment by DomT4** [comment]
Closing here. Will happily consider a PR rebased against master here and with an added test, Thanks!

**Pull request closed without merging**